### PR TITLE
N28: country-slice

### PR DESCRIPTION
**Pull request by anopara**
_Part of #852_

### CODE REVIEW

File: content/news/028/index.md [hunk 1]
```diff
@@ -31,6 +31,17 @@ Feel free to send PRs about your own projects!
...
 
 - [Game Updates](#game-updates)
+
+### Country Slice
+
+![country-slice-gif](country_slice_gif.gif)
+
+[Country Slice](https://github.com/anopara/country-slice) is [@anastasiaopara](https://twitter.com/anastasiaopara/)'s hobby project, where users can draw a small scene, and their input is amplified with real-time procedural geometry generation.
+
+Country Slice uses [Bevy Engine](https://github.com/bevyengine/bevy) for entity management, and has recently [been ported to OpenGL](https://twitter.com/anastasiaopara/status/1464304076074672144?s=20) (using [gl-rs](https://github.com/brendanzab/gl-rs/tree/master/gl) and [glutin](https://github.com/rust-windowing/glutin)). It is being developed openly on [GitHub](https://github.com/anopara/country-slice).
```
Comment (AngelOnFira) [inline content/news/028/index.md:41]: For any links you have, can you extract them to the bottom?

Example: https://github.com/rust-gamedev/rust-gamedev.github.io/pull/860/files

**Review by AngelOnFira — CHANGES_REQUESTED** [review]

**Comment by AngelOnFira** [comment]
Also looks like there may be some linting issues :)

**Comment by anopara** [comment]
Updated the links & broke down the lines (I think it should be <80 chars now 😅)

**Comment by AngelOnFira** [comment]
Oh wow! I didn't actually load the page yet, but I was very impressed by this project when I saw it on Twitter. Big fan of your work, I loved the texture synthesis posts you've written. Please keep adding content to the newsletter in the future! :smile:

**Review by AngelOnFira — APPROVED** [review]

**Comment by anopara** [comment]
Thank you @AngelOnFira 😃

**Pull request merged**